spec.md

```diff
@@ -200,6 +200,15 @@ that contains both context and data).
   * If present, MUST adhere to the format specified in
     [RFC 3986](https://tools.ietf.org/html/rfc3986)
 
+### content-type
+* Type: String per [RFC 2046](https://tools.ietf.org/html/rfc2046)
+* Description: Describe the data encoding format 
+* Constraints:
+  * OPTIONAL
+  * If present, MUST adhere to the format specified in
```

Comment:
Is there a default if this attribute is not present?

Reply:
its not different than schema-url, if you dont have it you need to figure out how to interpret the data/body yourself